src/Exception/InvalidUuidProvidedException.php

```diff
@@ -0,0 +1,8 @@
+<?php
+declare(strict_types=1);
+
+namespace MyOnlineStore\Common\Domain\Exception;
+
+final class InvalidUuidProvidedException extends \LogicException
```

Comment:
I would not use `\LogicException` as that indicates a flaw in program logic. Whether this case indicates an incorrect argument being supplied to one of the uuid instantiating functions. This will also causes a BC break and require a major version update. Using \InvalidArgumentException instead would avoid this

Reply:
you are correct! :+1:

Reply:
you are correct! :+1: